lib/ts/recipe/session/cookieAndHeaders.ts

```diff
@@ -146,7 +120,7 @@ export function setToken(
             tokenType === "refresh" ? "refreshTokenPath" : "accessTokenPath"
         );
     } else if (transferMethod === "header") {
-        setHeader(res, getHeaderNameFromTokenType(tokenType), value, expires);
+        setHeader(res, getResponseHeaderNameForTokenType(tokenType), value, expires);
```

Comment:
right below, the setHeader function has a `if (value === "remove") {` -> do we need that if block still?

Reply:
I guess we can remove it since we are not using that helper for the front-token